get/install.bash

```diff
@@ -2,37 +2,74 @@
 # webexec installation script
 #
 # This script is meant for quick & easy install via:
+#
+#   $ curl -L https://get.webexec.sh -o get-webexec.sh | bash get-webexec.sh
 set -x
-
-#   $ curl -L https://get.webexec.sh | bash
 SCRIPT_COMMIT_SHA=UNKNOWN
 LATEST_VERSION="0.15.3"
 
 # The latest release is currently hard-coded.
-echo "Installing " $LATEST_VERSION "version"
+echo "Installing webexec version " $LATEST_VERSION
          
-ARCH="$(uname -m)"  # -i is only linux, -m is linux and apple
-if [[ "$ARCH" = x86_64* ]]; then
+ARCH="$(uname -m | tr [:upper:] [:lower:])" 
+if [ "$ARCH" = x86_64* ]; then
     if [[ "$(uname -a)" = *ARM64* ]]; then
```

Comment:
why is ARM64 checked **inside** x86_64 ?
I don't have any ARM64 machine handy to check this myself.

Reply:
this is a special case for darwin m1 sh*t:

```
❯ uname -m
x86_64
❯ uname -a
Darwin Mac-mini.local 21.1.0 Darwin Kernel Version 21.1.0: Wed Oct 13 17:33:24 PDT 2021; root:xnu-8019.41.5~1/RELEASE_ARM64_T8101 x86_64
```